src/components/search/Search.jsx

```diff
@@ -8,18 +8,22 @@ const {default: Input, COLOR, SIZE} = InputModule;
 
 type PropsType = {
   adaptiveIco?: boolean,
-  className?: string,
-  ...InputModule.InputPropsType
-};
+  inputClassName?: string
+} & InputModule.InputPropsType; // TODO: make back to spread (...InputModule.InputPropsType) after flow bump
```

Comment:
why after? It was working before

Reply:
we investigate it with @PinkaminaDianePie it doesn't see inputClassName prop, strange behavior